Source/Atj/data/mvp_scenario.json

```diff
@@ -227,6 +241,66 @@
                     "routine": "debug_player_move_to_bulletin_board_1_routine"
                 }
             ]
+        },
+        {
+            "name": "increment_mood_juan_action",
+            "signals": [
+                {
+                    "type": "increment_mood",
+                    "npc": "juan",
+                    "value": 1
+                }
+            ]
+        },
+        {
+            "name": "increment_mood_zak_action",
+            "signals": [
+                {
+                    "type": "increment_mood",
+                    "npc": "zak",
+                    "value": 1
+                }
+            ]
+        },
+        {
+            "name": "increment_mood_sara_action",
+            "signals": [
+                {
+                    "type": "increment_mood",
+                    "npc": "sara",
+                    "value": 1
+                }
+            ]
+        },
+        {
+            "name": "decrement_mood_juan_action",
+            "signals": [
+                {
+                    "type": "increment_mood",
+                    "npc": "juan",
+                    "value": -1
+                }
+            ]
+        },
+        {
+            "name": "decrement_mood_sara_action",
+            "signals": [
+                {
+                    "type": "increment_mood",
+                    "npc": "sara",
+                    "value": -1
+                }
+            ]
+        },
+        {
+            "name": "decrement_mood_zak_action",
+            "signals": [
+                {
+                    "type": "increment_mood",
+                    "npc": "zal",
```

Comment:
Who is `zal`? 🤣 😆 🤣

Reply:
Currently my greatest scripting check-in failure to date!!!! Good eye my friend.